backend/src/scraper/consumer_reports_scraper.py

```diff
@@ -45,51 +46,46 @@ def scrape_expert_reviews(q: str, id_check: str, id_field: str) -> Dict:
-            er = review_response.json()
-        except:
-            er = {}
+        if ratio[0] in cleaned_title:
+            try:
+                review_url = "https://consumer-reports.p.rapidapi.com/products/detail"
+                review_querystring = {"id": id}
+                review_response = requests.get(review_url, headers=headers, params=review_querystring)
+                er = review_response.json()
+            except:
+                er = {}
 
-        if "message" in er.keys():
-            logging.error(f"Consumer Reports Product => Error: {er.get('message')}")
-            return {
-                "expert_review": {
-                    "product": q,
-                    "error": er.get("message"),
+            if "message" in er.keys():
+                logging.error(f"Consumer Reports Product => Error: {er.get('message')}")
+                return {
+                    "expert_review": {
+                        "product": q,
+                        "error": er.get("message"),
+                    }
                 }
-            }
-        elif len(er) == 0:
-            return {
-                "expert_review": {
-                    "product": q,
-                    "error": "The review for this product does not exist.",
+            elif len(er) == 0:
+                return {
+                    "expert_review": {
+                        "product": q,
+                        "error": "The review for this product does not exist.",
+                    }
                 }
-            }
-        else:
-            er = er.get('content')[0]
-            # Prepare to create link.
-            index = er.get('slugName').rfind('-')
-            name = er.get('slugName')[:index]
-            model = er.get('slugName')[index + 1:]
-            link = f"https://www.consumerreports.org/electronics-computers/cameras/{name}/m{model}/"
-            status = requests.get(link).status_code
-            if status != 200:
-                link = None
-
-            # Check that UPC or Walmart ID returned by Consumer Reports 
-            # matches UPC or Walmart ID returned by Best Buy or Walmart, respectively.
-            if er.get(id_field) is None:
-                field = ''
             else:
-                field = str(er.get(id_field))
+                er = er.get('content')[0]
+                # Prepare to create link.
+                index = er.get('slugName').rfind('-')
+                name = er.get('slugName')[:index]
+                model = er.get('slugName')[index + 1:]
+                link = f"https://www.consumerreports.org/electronics-computers/cameras/{name}/m{model}/"
+                headers = {"User-Agent": "Mozilla/5.0 (X11; CrOS x86_64 12871.102.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.141 Safari/537.36"}
+                status = requests.get(link, headers = headers).status_code
+                if status != 200:
+                    link = None
```

Comment:
Do you know how often a value is returned for `link`? When I was trying for a few products a few days ago, I typically wasn't getting anything.

The frontend expert reviews section doesn't even use `link` yet. Wondering if it's worth keeping this or not (if it is, I'll still need to implement this field on the frontend)

Reply:
Pretty much every single time. I think nothing was being returned before because the request was returning a 403 code instead of 200. But now that that's been fixed, I've been seeing a link pretty frequently.

Reply:
Oh okay nice! I'll add a ticket for myself to implement this then